care/facility/migrations/0483_migrate_patient_consultations.py

```
@@ -0,0 +1,827 @@
+        return None
+    text = ", ".join(investigation["type"])
+    if investigation.get("repetitive", False) and investigation.get("frequency"):
+        text += f", Every {investigation['frequency']}"
+    if time := investigation.get("time"):
+        time.replace("T", " ")
+        text += f", At {time}"
+    if notes := investigation.get("notes"):
+        text += f"\nNotes: {notes}"
+
+
+def migrate_consultations(apps, schema_editor):
```

Ruff (0.8.2)

Too many branches (48 > 12)

(PLR0912)